### PR TITLE
Fix for geo feature coverage display

### PR DESCRIPTION
**Pull request by pkdash**
### Pull Request Checklist: 
- [x] Positive Test Case Written by Dev

- [x] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [x] Passing Jenkins Build

- [x] Peer Code review and approval

### Positive Test Case
1. Create a composite resource. 
- Upload a zip file that contains files for a geo feature dataset that has point type coverage to this composite resource. Unzip the uploaded file, This should create geo feature aggregation. Select the aggregation to see metadata for the aggregation. Verify the 'Point' type coverage is displayed.
- Upload another zip file that contains files for a geo feature dataset that has box type coverage to this composite resource. Unzip the uploaded file, This should create geo feature aggregation. Select the aggregation to see metadata for the aggregation. Verify the 'Box' type coverage is displayed.

### CODE REVIEW

File: theme/static/js/resourceLandingAjax.js [hunk 1]
```diff
@@ -1774,30 +1774,45 @@ function updateEditCoverageStateFileType() {
...
     if (logical_type !== "GenericLogicalFile" && logical_type !== "FileSetLogicalFile"){
-        // don't allow changing coverage type
+        // don't allow changing coverage type if aggregation type is not GenericLogicalFile or FileSetLogicalFile
         $id_type_filetype_div.parent().closest("div").css('pointer-events', 'none');
         $id_type_filetype_div.find(radioBoxSelector).attr('onclick', 'return false');
         $id_type_filetype_div.find(radioPointSelector).attr('onclick', 'return false');
-        if (logical_type !== "RefTimeseriesLogicalFile"){
-            $id_type_filetype_div.find(radioBoxSelector).attr('checked', 'checked');
+
+        var selectBoxTypeCoverage = false;
```
Comment (Maurier) [inline theme/static/js/resourceLandingAjax.js]: Good use of boolean variable. Makes the code easier to understand.

**Comment by pkdash** [comment]
@dtarb Let me know if you want to test this fix (#3158) before I merge this PR. I can deploy this branch for you to test.

**Comment by dtarb** [comment]
If you would like to deploy, I would be happy to test.  However if you are confident in the fix, I am happy for you to merge.

**Comment by pkdash** [comment]
@dtarb This is all JavaScript code changes. That's why I am not as confident as I should be. Will deploy and let you know.

**Comment by cuahsimarko** [comment]
I'll be recommending a long-term solution to @phuongdm that helps out with these kinds of changes, making it an afterthought to be able to get Neal, Liza, David, Jeff, Martin able to quickly view a change, without a lengthy or disruptive deployment. We'll see if it works out, stay tuned!

**Comment by dtarb** [comment]
OK.   Happy to look.  Let me know when it is ready.

**Comment by cuahsimarko** [comment]
Yep after Phu and I discuss if it gets implemented he can present it on a Mon/Wed call or at least send out an e-mail.

**Comment by pkdash** [comment]
@dtarb I was not able to deploy. The deployment build fails due to recent configuration changes. May be Jenkins build script needs to change to cope with code base changes. I guess I am going to merge this PR today and hope for the best. If you find this issue has not been fixed during beta testing, I will make another attempt to fix it that time.

**Comment by dtarb** [comment]
@pkdash thanks for trying.  Which platform were you trying on.  @cuahsimarko @sblack-usu @cbcunc please check into why Pabitra can not deploy on a test platform.

**Comment by pkdash** [comment]
@dtarb I was trying to deploy on playground. But I don't think it is an issue with deploy target machine.

**Comment by dtarb** [comment]
@pkdash can you clarify what code base changes are causing the deployment build to fail.  I plan to raise this on the Monday call.  I know that there has been work to "improve" things, but if improving is breaking and blocking this work then we need to discuss.

**Comment by pkdash** [comment]
@dtarb The codebase no more has a file called 'local_settings.py'. This is a recent change. However, at the container build time this file is expected.  My guess is that the file 'local_settings.cutover' needs to be renamed as 'local_settings.py' as part of the build script (hsctl).

**Comment by phuongdm** [comment]
@pkdash ,

I've fixed the Jenkins job (just knew about this job), there were 3 problems:
1. local_settings.py
2. Too old version of docker (docker-io, not docker-ce)
3. No space left after several upgrade of hs_docker_base from @sblack-usu 

Please deploy your branch and let me know if any other issue

Thanks,
Phuong

**Comment by pkdash** [comment]
@phuongdm Thanks. I just started the deployment.

**Comment by pkdash** [comment]
@dtarb It is now deployed on playground for you to test this issue. I will wait to merge until you are done with testing.

**Comment by dtarb** [comment]
Works perfectly on playground.  Thanks @pkdash and @phuongdm

**Pull request merged**